src/Illuminate/Support/Arr.php

```diff
@@ -100,6 +100,23 @@ public static function dot($array, $prepend = '')
     }
 
     /**
+     * Expand dot-flattened array into a multi-dimensional associative array
```

Comment:
Explain here, please.
It is the same as [here](https://github.com/laravel/framework/blob/5.1/src/Illuminate/Support/Arr.php#L81), isn't it?